src/test/java/underdogs/devbie/acceptance/AcceptanceTest.java

```diff
@@ -9,40 +9,55 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import io.restassured.RestAssured;
 import io.restassured.specification.RequestSpecification;
 import underdogs.devbie.auth.dto.UserTokenDto;
 import underdogs.devbie.auth.jwt.JwtTokenProvider;
 import underdogs.devbie.user.domain.User;
+import underdogs.devbie.user.dto.UserCreateRequest;
 
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 public abstract class AcceptanceTest {
 
+    protected final ObjectMapper objectMapper = new ObjectMapper();
+
     @LocalServerPort
     protected int port;
 
-    private static RequestSpecification given() {
-        return RestAssured.given().log().all();
-    }
+    protected String bearerToken;
 
     @Value("${security.jwt.token.secret-key:sample}")
     private String secret;
 
-    @Value("${security.jwt.token.expire-length:300}")
+    @Value("${security.jwt.token.expire-length:300000}")
     private long seconds;
 
-    protected String bearerToken;
+    private static RequestSpecification given() {
+        return RestAssured.given().log().all();
+    }
 
     @BeforeEach
-    void setUp() {
+    void setUp() throws JsonProcessingException {
         RestAssured.port = port;
-        UserTokenDto userTokenDto = UserTokenDto.from(User.builder().id(1L).build());
+
+        Long userId = createUser();
+        UserTokenDto userTokenDto = UserTokenDto.from(User.builder().id(userId).build());
```

Comment:
여기 빌더 패턴도 아래와 같이 포맷을 맞추면 좋겠네요 :)
```suggestion
        UserTokenDto userTokenDto = UserTokenDto.from(User.builder()
               .id(userId)
               .build());
```

Reply:
매의 눈이시군요 👍